### PR TITLE
Expose /_system handler

### PR DESCRIPTION
**Pull request by kxepal**

### CODE REVIEW

File: src/chttpd_httpd_handlers.erl [hunk 1]
```diff
@@ -25,6 +25,7 @@ url_handler(<<"_replicate">>)      -> fun chttpd_misc:handle_replicate_req/1;
 url_handler(<<"_uuids">>)          -> fun chttpd_misc:handle_uuids_req/1;
 url_handler(<<"_session">>)        -> fun chttpd_auth:handle_session_req/1;
 url_handler(<<"_up">>)             -> fun chttpd_misc:handle_up_req/1;
+url_handler(<<"_system">>)         -> fun chttpd_misc:handle_system_req/1;
```
Comment (rnewson) [inline src/chttpd_httpd_handlers.erl]: we don't want this here, we want a 404.

**Comment by rnewson** [comment]
+1 after removing /_system. We want that to 404, the correct method is /_node/<fqdn>/_system as this works on any node for any node.

File: src/chttpd_httpd_handlers.erl (hunk 1 above)
Comment (kxepal) [inline src/chttpd_httpd_handlers.erl]: ok, fixed

**Comment by rnewson** [comment]
+1. I agree we should move this handle to couch/src/couch_httpd_ instead, but not blocking the release.

**Comment by kxepal** [comment]
Ok, will move it later with next iteration.

**Comment by kxepal** [comment]
Closed with 936e8ee

**Pull request closed without merging**